l10n_it_withholding_tax/models/withholding_tax.py

```diff
@@ -76,6 +76,15 @@ def _default_wt_journal(self):
     )
     daticassprev_tax_id = fields.Many2one('account.tax')
 
+    @api.multi
+    def copy(self, default=None):
+        self.ensure_one()
+        default = dict(default or {})
+        if default.get("code", False):
```

Comment:
```suggestion
        if "code" not in default:
            default["code"] = _("%s (copy)") % self.code
        return super(WithholdingTax, self).copy(default)
```
In questo modo hai un unico `return` all'interno del metodo rendendolo così più _clean_. Che ne pensi?

Reply:
Modificato

Reply:
Grazie! Ultima cosa: `or ""` è necessario?

Reply:
mah, no, modifico